### PR TITLE
Added the Campaign

### PR DESCRIPTION
**Pull request by infosec-alchemist**

### CODE REVIEW

File: cti-stix-ui/app/models/campaign.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ * @extends ember-data/Model
+ */
+export default DS.Model.extend({
+    name: DS.attr("string"),
+    description: DS.attr(),
+    labels: DS.attr(),
+    aliases: DS.attr(),
+    external_references: DS.attr(),
+    version: DS.attr("string"),
+    created: DS.attr("Date"),
```
Comment (exceptionfactory) [inline cti-stix-ui/app/models/campaign.js]: Although DS.attr() may accept the capitalized word Date, all type identifiers should be lowercased (http://emberjs.com/api/data/classes/DS.html#method_attr).

File: cti-stix-ui/app/models/course-of-action.js [hunk 2]
```diff
@@ -13,8 +13,8 @@ export default DS.Model.extend({
     labels: DS.attr(),
     external_references: DS.attr(),
     version: DS.attr("string"),
-    created: DS.attr("string"),
-    modified: DS.attr("string"),
+    created: DS.attr("Date"),
+    modified: DS.attr("Date"),
```
Comment (exceptionfactory) [inline cti-stix-ui/app/models/course-of-action.js]: Although DS.attr() may accept the capitalized word Date, all type identifiers should be lowercased (http://emberjs.com/api/data/classes/DS.html#method_attr).

File: cti-stix-ui/app/templates/campaign.hbs [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+      <div class="col">
+        <p>Description</p>
+        <p>{{model.item.description}}</p>
+      </div>
+    </div>
+  {{#if model.item.first_seen}}
+    <div class="row">
+      <div class="col">
+        <p>First Seen</p>
+            <span class="published"><p>{{moment-format report.first_seen "YYYY-MM-DD HH:MM"}}</p></span>
```
Comment (exceptionfactory) [inline cti-stix-ui/app/templates/campaign.hbs]: The moment-format argument should be model.item.first_seen, not report.first_seen.

File: cti-stix-ui/app/models/campaign.js (hunk 1 above)
Comment (infosec-alchemist) [inline cti-stix-ui/app/models/campaign.js]: fixed

File: cti-stix-ui/app/models/course-of-action.js (hunk 2 above)
Comment (infosec-alchemist) [inline cti-stix-ui/app/models/course-of-action.js]: fixed

File: cti-stix-ui/app/templates/campaign.hbs (hunk 3 above)
Comment (infosec-alchemist) [inline cti-stix-ui/app/templates/campaign.hbs]: Copy and paste, but it still worked, which seems weird.

**Pull request merged**